features/support/capybara.rb

```diff
@@ -7,6 +7,7 @@
 WINDOW_SIZE_ARG = '--window-size=1280,2048'
 
 Capybara.configure do |config|
+  config.default_driver = :rack_test
```

Comment:
This actually doesn't do anything, the default driver was `:rack_test` anyway. This is just for clarity, because I am using `Capybara.default_driver` in a different place, and I think it's better make it explicit (rather than making the next developer go to Capybara docs to figure out the default).